File: VB/LocalizationSample/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace LocalizationSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()

'            #Region "#frlocalize"
            spreadsheetControl1.Options.Behavior.FunctionNameCulture = DevExpress.XtraSpreadsheet.FunctionNameCulture.Local
            spreadsheetControl1.Options.Culture = New System.Globalization.CultureInfo("fr-FR")
'            #End Region ' #frlocalize
        End Sub
    End Class
End Namespace

File: CS/LocalizationSample/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace LocalizationSample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();

            #region #frlocalize
            spreadsheetControl1.Options.Behavior.FunctionNameCulture = DevExpress.XtraSpreadsheet.FunctionNameCulture.Local;
            spreadsheetControl1.Options.Culture = new System.Globalization.CultureInfo("fr-FR");
            #endregion #frlocalize
        }
    }
}
